runtime/compiler/control/J9Options.cpp

```diff
@@ -1079,11 +1079,11 @@ static std::string readFileToString(char *fileName)
    int readSize = 0;
    int iter = 0;
    do {
-      readSize = j9jit_fread(fileId, buf, BUFFER_SIZE);
+      readSize = fread(buf, BUFFER_SIZE, 1, f->_stream);
       fileStr.append(buf, readSize);
       ++iter;
    } while ((readSize == BUFFER_SIZE) && (iter < MAX_FILE_SIZE_IN_PAGES));
-   j9jit_fcloseId(fileId);
+   j9jit_fclose(f);
 
    if (iter < MAX_FILE_SIZE_IN_PAGES)
```

Comment:
Here, I think we need to use `<=`, because if we set `MAX_FILE_SIZE_IN_PAGES` to 1 we will always return an empty string.